Docker build in (pre)release workflow broken: error: cannot run C compiled programs

STR:
- Tag a pre-release (2.0.0-beta.1)

Expected:
- GitHub Actions prerelease workflow successfully builds and pushes new Docker images for three different architectures

Actual:
- GitHub prerelease action fails with the following error:

```
#24 490.5 npm error path /home/node/webthings/gateway/node_modules/jpegtran-bin
#24 490.5 npm error command failed
#24 490.5 npm error command sh -c node lib/install.js
#24 490.5 npm error ⚠ spawn /home/node/webthings/gateway/node_modules/jpegtran-bin/vendor/jpegtran ENOENT
#24 490.5 npm error   ⚠ jpegtran pre-build test failed
#24 490.5 npm error   ℹ compiling from source
#24 490.5 npm error   ✖ Error: Command failed: /bin/sh -c ./configure --disable-shared --prefix="/home/node/webthings/gateway/node_modules/jpegtran-bin/vendor" --bindir="/home/node/webthings/gateway/node_modules/jpegtran-bin/vendor"
#24 490.5 npm error configure: error: in `/tmp/7bc78126-0f87-4751-ae3a-9ca8589b297b':
#24 490.5 npm error configure: error: cannot run C compiled programs.
#24 490.5 npm error If you meant to cross compile, use `--host'.
#24 490.5 npm error See `config.log' for more details
#24 490.5 npm error
```

https://github.com/WebThingsIO/gateway/actions/runs/16448773663/job/46487971770

This appears to relate to (cross?) compiling the jpegtran-bin npm dependency.

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -30,18 +30,29 @@ jobs:
   build-docker-images:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout repository
+        uses: actions/checkout@v4
       - name: Set release version
         run: echo "RELEASE_VERSION=${GITHUB_REF:10}" >> $GITHUB_ENV
-      - name: Build image
-        uses: ilteoood/docker_buildx@master
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
         with:
-          publish: true
-          imageName: webthingsio/gateway
-          platform: linux/amd64,linux/arm/v7,linux/arm64
-          tag: latest,${{ env.RELEASE_VERSION }}
-          dockerUser: ${{ secrets.DOCKER_HUB_USER }}
-          dockerPassword: ${{ secrets.DOCKER_HUB_PASSWORD }}
+          username: ${{ secrets.DOCKER_HUB_USER }}
+          password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+      - name: Build and push Docker image (multi-arch)
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms: linux/amd64,linux/arm/v7,linux/arm64
+          push: true
+          tags: webthingsio/gateway:${{ env.RELEASE_VERSION }}
+      - name: Clean up Docker
+        run: docker system prune -af
 
   build-raspbian-image:
     needs: create-release

